### ISSUE TITLE
Not a single ggml bin file will load.

### ISSUE THREAD

**Issue by CRCODE22** [oobabooga/text-generation-webui#1406]
### Describe the bug

Not a single ggml bin file will load

I am using the latest of "D:\one-click-installers\text-generation-webui\repositories\GPTQ-for-LLaMa" also I have manually build the cuda without any errors.

Here is two examples of bin files that will not work:

OSError: It looks like the config file at ‘models\ggml-vicuna-13b-4bit-rev1.bin’ is not a valid JSON file
OSError: It looks like the config file at ‘models\ggml-vicuna-7b-1.0-uncensored-q4_2.bin’ is not a valid JSON file.

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Reproduction

call python server.py --chat --wbits 4 --groupsize 128 --gpu-memory 10737418240 --cpu-memory 10GB --auto-devices

Then select the models manually from the models menu.

### Logs

```shell
Traceback (most recent call last):
File “D:\one-click-installers\installer_files\env\lib\site-packages\transformers\configuration_utils.py”, line 658, in _get_config_dict
config_dict = cls._dict_from_json_file(resolved_config_file)
File “D:\one-click-installers\installer_files\env\lib\site-packages\transformers\configuration_utils.py”, line 745, in _dict_from_json_file
text = reader.read()
File “D:\one-click-installers\installer_files\env\lib\codecs.py”, line 322, in decode
(result, consumed) = self._buffer_decode(data, self.errors, final)
UnicodeDecodeError: ‘utf-8’ codec can’t decode byte 0x80 in position 28: invalid start byte

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
File “D:\one-click-installers\text-generation-webui\server.py”, line 100, in load_model_wrapper
shared.model, shared.tokenizer = load_model(shared.model_name)
File “D:\one-click-installers\text-generation-webui\modules\models.py”, line 186, in load_model
model = LoaderClass.from_pretrained(checkpoint, **params)
File “D:\one-click-installers\installer_files\env\lib\site-packages\transformers\models\auto\auto_factory.py”, line 441, in from_pretrained
config, kwargs = AutoConfig.from_pretrained(
File “D:\one-click-installers\installer_files\env\lib\site-packages\transformers\models\auto\configuration_auto.py”, line 916, in from_pretrained
config_dict, unused_kwargs = PretrainedConfig.get_config_dict(pretrained_model_name_or_path, **kwargs)
File “D:\one-click-installers\installer_files\env\lib\site-packages\transformers\configuration_utils.py”, line 573, in get_config_dict
config_dict, kwargs = cls._get_config_dict(pretrained_model_name_or_path, **kwargs)
File “D:\one-click-installers\installer_files\env\lib\site-packages\transformers\configuration_utils.py”, line 661, in _get_config_dict
raise EnvironmentError(
OSError: It looks like the config file at ‘models\ggml-vicuna-7b-1.0-uncensored-q4_2.bin’ is not a valid JSON file.
```

### System Info

```shell
32GB RAM Windows 11 64 Bits Nvidia 3060 12GB Gaming OC 2.0.
```

**Comment by oobabooga** [comment]
https://github.com/oobabooga/text-generation-webui/wiki/llama.cpp-models#using-llamacpp-in-the-web-ui

**Comment by yhyu13** [comment]
@oobabooga https://github.com/oobabooga/text-generation-webui/blob/main/docs/llama.cpp-models.md

I am having the same problem as @CRCODE22, we have placed the weight as `models\ggml-vicuna-7b-1.0-uncensored-q4_2.bin`.

This is exactly what the doc suggests:
> Pre-converted models
> Simply place the model in the models folder, making sure that its name contains ggml somewhere and ends in .bin.

**Comment by yhyu13** [comment]
I am going to try this : https://github.com/oobabooga/text-generation-webui/issues/1248#issuecomment-1510437316

**Comment by yhyu13** [comment]
https://github.com/oobabooga/text-generation-webui/issues/1248#issuecomment-1510437316
does not work either!
```
(textgen) hangyu5@hangyu5deMac-mini text-generation-webui % python server.py --model ggml-vicuna-13b-1.1-q4_1.bin --threads 8
Gradio HTTP request redirected to localhost :)
bin /Users/hangyu5/anaconda3/envs/textgen/lib/python3.10/site-packages/bitsandbytes/libbitsandbytes_cpu.so
/Users/hangyu5/anaconda3/envs/textgen/lib/python3.10/site-packages/bitsandbytes/cextension.py:33: UserWarning: The installed version of bitsandbytes was compiled without GPU support. 8-bit optimizers, 8-bit multiplication, and GPU quantization are unavailable.
  warn("The installed version of bitsandbytes was compiled without GPU support. "
Loading ggml-vicuna-13b-1.1-q4_1.bin...



Traceback (most recent call last):
  File "/Users/hangyu5/anaconda3/envs/textgen/lib/python3.10/site-packages/transformers/configuration_utils.py", line 658, in _get_config_dict
    config_dict = cls._dict_from_json_file(resolved_config_file)
  File "/Users/hangyu5/anaconda3/envs/textgen/lib/python3.10/site-packages/transformers/configuration_utils.py", line 745, in _dict_from_json_file
    text = reader.read()
  File "/Users/hangyu5/anaconda3/envs/textgen/lib/python3.10/codecs.py", line 322, in decode
    (result, consumed) = self._buffer_decode(data, self.errors, final)
UnicodeDecodeError: 'utf-8' codec can't decode byte 0x80 in position 28: invalid start byte

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/hangyu5/Documents/git-repoMy/AIResearchVault/repo/LLMApp/text-generation-webui/server.py", line 921, in <module>
    shared.model, shared.tokenizer = load_model(shared.model_name)
  File "/Users/hangyu5/Documents/git-repoMy/AIResearchVault/repo/LLMApp/text-generation-webui/modules/models.py", line 56, in load_model
    model = LoaderClass.from_pretrained(Path(f"{shared.args.model_dir}/{model_name}"), low_cpu_mem_usage=True, torch_dtype=torch.bfloat16 if shared.args.bf16 else torch.float16, trust_remote_code=trust_remote_code)
  File "/Users/hangyu5/anaconda3/envs/textgen/lib/python3.10/site-packages/transformers/models/auto/auto_factory.py", line 441, in from_pretrained
    config, kwargs = AutoConfig.from_pretrained(
  File "/Users/hangyu5/anaconda3/envs/textgen/lib/python3.10/site-packages/transformers/models/auto/configuration_auto.py", line 916, in from_pretrained
    config_dict, unused_kwargs = PretrainedConfig.get_config_dict(pretrained_model_name_or_path, **kwargs)
  File "/Users/hangyu5/anaconda3/envs/textgen/lib/python3.10/site-packages/transformers/configuration_utils.py", line 573, in get_config_dict
    config_dict, kwargs = cls._get_config_dict(pretrained_model_name_or_path, **kwargs)
  File "/Users/hangyu5/anaconda3/envs/textgen/lib/python3.10/site-packages/transformers/configuration_utils.py", line 661, in _get_config_dict
    raise EnvironmentError(
OSError: It looks like the config file at 'models/ggml-vicuna-13b-1.1-q4_1.bin' is not a valid JSON file.
```

**Comment by yhyu13** [comment]
@CRCODE22  ggml can now be loaded. fixed in 2f6e2ddeac28090c06c9c2cc14bce971dba70ffd
